packages/DataGrid/stories/DataGrid.zebra.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import * as Sbook from "storybook/assets/styles/common.styles";
+import worker from "workerize-loader!./helpers/data.worker"; // eslint-disable-line import/no-webpack-loader-syntax
+import Spinner from "@paprika/spinner";
+import DataGrid from "../src";
+
+export function App() {
+  const [data, setData] = React.useState([]);
+  const [isIdle, setIsIdle] = React.useState(true);
```

Comment:
`isLoading`?  Clearer to use common language here between other components terminology and the `loadData` function.